_site/public/docs/getting_started/client.md

```diff
@@ -9,7 +9,7 @@ When developing your Truffle-based application, we recommend using the [Ethereum
 # When Deploying to Live Networks
 
 * Geth (go-ethereum): [https://github.com/ethereum/go-ethereum](https://github.com/ethereum/go-ethereum)
-* WebThree (cpp-ethereum): [https://github.com/ethereum/webthree-umbrella](https://github.com/ethereum/webthree-umbrella)
+* WebThree (cpp-ethereum): [https://github.com/ethereum/cpp-ethereum](https://github.com/ethereum/cpp-ethereum)
```

Comment:
Instead of WebThree it should have name like Ethereum C++ client or something else